docs/copilot/copilot-coding-agent.md

```diff
@@ -0,0 +1,227 @@
+
+1. In the **GitHub Pull Requests** view, navigate to the **Issues** section
+
+1. Find the issue you want to assign to Copilot
+
+1. Right-click the issue and select **Assign to Copilot** or click the assign button and select `@copilot`
+
+1. The agent will begin working on the issue in the background
+
+   ![Screenshot showing the GitHub Pull Requests view, highlighting the assign to Copilot action, and the PR query for work assigned to Copilot.](images/copilot-coding-agent/github-pull-request-coding-agent.png)
+
+### Method 2: Delegate from Copilot Chat
+
+1. Open Copilot Chat (`kb(workbench.action.chat.open)`)
+
+1. Have a conversation about the feature or change you want to implement
+
+1. When ready, delegate to the agent using one of these methods:
+
+   **Use the delegate button (Experimental)**
+
+   Enable the experimental setting `setting(githubPullRequests.codingAgent.uiIntegration)` to show a **Delegate to coding agent** button in the Chat view for repositories that have the agent enabled. Select this button to hand off your current chat context to the coding agent.
+
+   <video src="images/copilot-coding-agent/delegate-to-coding-agent.mp4" title="Video showing how to delegate to coding agent from VS Code chat." controls poster="images/copilot-coding-agent/delegate-to-coding-agent-poster.png"></video>
+
+   **Use the #copilotCodingAgent tool**
+
+   You can also reference the tool directly in your prompt to ask Copilot to continue a local change in the background. This tool automatically pushes pending changes to a remote branch and initiates a coding agent session:
+
+   ![Screenshot showing handing off a session to Copilot coding agent](images/copilot-coding-agent/coding-agent-start.png)
+
+1. The agent will create a pull request and begin implementing the discussed changes
+
+## Tracking agent progress
+
+### Monitor work in VS Code
+
+The GitHub Pull Requests extension provides a dedicated **Copilot on My Behalf** section that shows:
+
+* All active Copilot coding agent sessions
+* Pull requests created by the agent
+* Progress status for each task
+* Numeric badges indicating new changes or updates
+
+![Screenshot showing status of multiple coding agent pull requests](images/copilot-coding-agent/coding-agent-status.png)
+
+### View detailed session logs
+
+1. In the Pull Requests view, find your agent's work under **Copilot on My Behalf**
+
+1. Select **View Session** to see a detailed log of everything the agent did:
+   * Commands executed
+   * Files modified
+   * Tests run
+   * Decision-making process
+
+   ![Screenshot showing the session log of a coding agent session.](images/copilot-coding-agent/coding-agent-session-log.png)
+
+> [!TIP]
+> When working with pull requests created by the coding agent, the `#activePullRequest` tool is automatically attached to chat, allowing you to maintain context and continue working on the pull request with access to the coding agent session information.
```

Comment:
This is not entirely clear to me. Can you elaborate which scenarios this enables? Does this mean I can intervene on the PR, or do work on the side and the coding agent picks up my changes?